control_msgs/msg/PidState.msg

```diff
@@ -0,0 +1,13 @@
+Header header
+float64 error
+float64 error_dot
+float64 dt
```

Comment:
Would it make more sense to make this a duration?.
Also, how about renaming `dt` to `timestep`?